packages/listview/src/components/PercentColumn/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useLocale from 'react-rainbow-components/libs/hooks/useLocale';
```

Comment:
we can't do this anymore, I have refactored all this imports because this causes issues with Next JS and SSR
In this case now you can use ```import { useLocale } from 'react-rainbow-components';```